TypeCobol/Tools/APIHelpers/Helpers.cs

```diff
@@ -140,8 +140,14 @@ public static SymbolTable LoadIntrinsic(List<string> paths, DocumentFormat intri
 #endif
                 try
                 {
+                    TypeCobolOptions options = new TypeCobolOptions(config);
+                    options.ExecToStep = ExecutionStep.SemanticCheck;
+
+#if EUROINFO_RULES
+                    options.AutoRemarksEnable = config.AutoRemarks;
```

Comment:
Seems it should be handled in the constructor of TypeCobolOptions.

Reply:
I made the same proposition in one of my previous PR, but @smedilol prefered to leave it as it is.
See https://github.com/TypeCobolTeam/TypeCobol/pull/1117#pullrequestreview-155963904

Reply:
The idea is not to add a new constructor (as in #1117) : the assignment of AutoRemarksEnable should be done in `public TypeCobolOptions(TypeCobolConfiguration config)` instead of outside the class.

Reply:
As spoken, I removed those property write in the parser code. The only occurences of these lines are in the test projects, where we manually set them.